doc/source/pipeflow/pipeflow_procedure.rst

```diff
@@ -98,8 +98,7 @@ The functions used to create the internal pit and extract results back from it a
 
 .. autofunction:: pandapipes.pf.pipeflow_setup.reduce_pit
 
-.. autofunction:: pandapipes.extract_results_active_pit
-   :noindex:
```

Comment:
Why was :noindex: removed? It was necessary because the function is documented twice